kostyor/tests/unit/rest_api/test.py

```diff
@@ -339,3 +340,61 @@ def test_discover_cluster_discovery_failed_error_response(self,
                                               auth_url='http://9.9.9.9')
         fake_session.assert_called_once_with(auth=password_mock)
         fake_discover.assert_called_once_with()
+
+    @mock.patch.object(db_api, 'get_cluster', mock.Mock())
+    @mock.patch.object(db_api, 'get_hosts_by_cluster', mock.Mock())
+    def test_host_list_cluster_exists_success(self):
+        fake_hosts = ['hostname_1', 'hostname_2']
```

Comment:
Also, I'm a bit concerned that (it's a topic for separate patch, obviously, but I want to express this concern here) once `get_hosts_by_cluster` is changed and start returning another values (not a list) we won't catch this change in this test, since it fully mocks restuls. Hence, all tests are passed but nothing is working in production. 

I think we should either provide integration (rest_api + db_layer) tests on early stages, or stop mocking dbapi and prepare objects for rest api in place. What do you think guys?